### PR TITLE
docs: routing

### PR DESCRIPTION
**Pull request by wkillerud**
Howdy 🤠 

I planned on tinkering some with Hubro today, but got a bit stuck deciphering some details about the routing. [The examples](https://github.com/hubro-apparatus/examples) are helpful, but I think future me will appreciate having a "one-pager" on how to use the router.

### CODE REVIEW

File: docs/2-installation.md [hunk 1]
```diff
@@ -41,9 +42,14 @@ Then add the following commands to the `scripts` field in the package.json in yo
 
 ## Advanced
```
Comment (wkillerud) [inline docs/2-installation.md:43]: This section can probably be extracted to a "deploy" chapter later on.

File: docs/4-routing.md [hunk 2]
```diff
@@ -1 +1,161 @@
...
+
+Hubro's file-based router looks for `page.js` files inside each subdirectory of `/pages`.
+
+- `/pages`
+  - `/about`
+    - `page.js`
+  - `page.js`
+
+Pages must include a function as its default export. That function must return
+a Lit `TemplateResult` (typically [html](https://lit.dev/docs/api/static-html/#html)).
```
Comment (wkillerud) [inline docs/4-routing.md:21]: The examples I could find all did this, but I'm not sure if it's strictly required to return a `TemplateResult` here.

**Comment by trygve-lie** [comment]
Man! Awesome. Thanks a lot!

File: docs/2-installation.md (hunk 1 above)
Comment (trygve-lie) [reply docs/2-installation.md:43]: Yeah. Definetly.

**Pull request merged**